contracts/registry/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "loam-registry"
```

Comment:
Is this named `loam-registry` or `Smart Deploy`? The README says SmartDeploy

Reply:
Good point!